### ISSUE TITLE
Please add COSANTA algo. I know you want to

### ISSUE THREAD

**Issue by dastardlyman12** [xmrig/xmrig#2923]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Required data**
 - Miner log as text or screenshot
 - Config file or command line (without wallets)
 - OS: [e.g. Windows]
 - For GPU related issues: information about GPUs and driver version.

**Additional context**

**Comment by dastardlyman12** [comment]
Cosanta - is the CPU algo we all want. oooh yes. suits me sir.

**Comment by Spudz76** [comment]
Seems like modified `X18` algo called `cosa`, could maybe port from their [example `cpuminer-multi` fork](https://github.com/cosanta/cpuminer-multi).

**Comment by SChernykh** [comment]
> we all want

Don't speak for everyone please. X18 something? So it's a huge pile of work for a coin with $1000 daily emission and unclear future, no thanks.

**Comment by gades** [comment]
COSA also is X17 + GHOST + LYRA2

**Comment by Asghan86** [comment]
> we all want.
No. Don't speak for me. I do not need it. therefore it is not "all"...
